src/MiscellaneousCopulas/RafteryCopula.jl

```diff
@@ -0,0 +1,105 @@
+"""
+    RafteryCopula{d, P}
+
+Fields:
+  - θ::Real - parameter
+
+Constructor
+
+    RafteryCopula(d, θ)
+
+The Multivariate Raftery Copula of dimension d is arameterized by ``\\theta \\in [0,1]`` 
+
+```math
+C_{\\theta}(\\mathbf{u}) = u_{(1)} + \\frac{(1 - \\theta)(1 - d)}{1 - \\theta - d} \\left(\\prod_{j=1}^{d} u_j\\right)^{\\frac{1}{1-\\theta}} - \\sum_{i=2}^{d} \\frac{\\theta(1-\\theta)}{(1-\\theta-i)(2-\\theta-i)} \\left(\\prod_{j=1}^{i-1}u_{(j)}\\right)^{\\frac{1}{1-\\theta}}u_{(i)}^{\\frac{2-\\theta-i}{1-\\theta}}
+```
+
+
+where ``u_{(1)}, \\ldots , u_{(d)}`` denote the order statistics of ``u_1, \\ldots ,u_d``.
+
+More details about Multivariate Raftery Copula are found in :
+
+    Saali, T., M. Mesfioui, and A. Shabri, 2023: Multivariate Extension of Raftery Copula. Mathematics, 11, 414, https://doi.org/10.3390/math11020414.    
+    
+    Nelsen, Roger B. An introduction to copulas. Springer, 2006. Exercise 3.6.
+
+It has a few special cases:
+- When θ = 0, it is the IndependentCopula.
+- When θ = 1, it is the the Fréchet upper bound
+"""
+struct RafteryCopula{d, P} <: Copula{d}
+    θ::P  # Copula parameter
+    function RafteryCopula(d,θ)
+        if (θ < 0) || (θ > 1)
+            throw(ArgumentError("Theta must be in [0,1]"))
+        elseif θ == 0
+            return IndependentCopula(d)
+        elseif θ == 1
+            return MCopula(d)
+        else
+            return new{d,typeof(θ)}(θ)
+        end
+    end
+end
+function _cdf(R::RafteryCopula, u::Vector{T}) where {T}
+    if length(u) != R.d
```

Comment:
R.d does not exists here. You may access the dimension of the copula using : 

```
function _cdf(R::RafteryCopula{d,P}, u::Vector{T}) where {d,P,T}
```

or by using `d = length(R)`, but `R.d` will not work. 

Moreover, there is no need to check the dimension of u this is done automatically now :)